### PR TITLE
1794/fix/identification-number-field-length

### PR DESCRIPTION
**Pull request by Noam6421**
Fix to Bug: [1794](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1794/)

Now if chosen identificationtype is of type id, the  identificationNumber can only be 9 numbers.

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningPersonal.tsx [hunk 1]
```diff
@@ -60,6 +62,11 @@ const ContactQuestioningPersonal: React.FC<Props> = (props: Props): JSX.Element
         if (watchIdentificationType || watchIdentificationNumber){
             trigger(identificationTypeFieldName);
             trigger(identificationNumberFieldName); 
+            if (watchIdentificationType === IdentificationTypesCodes.PALESTINE_ID || watchIdentificationType === IdentificationTypesCodes.ID) {
```
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningPersonal.tsx:65]: you can put the if statement inside the setIsId

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx [hunk 2]
```diff
@@ -52,6 +55,11 @@ const ContactForm: React.FC<Props> = ({ updatedContactIndex, contactStatus, pers
             trigger(identificationTypeFieldName);
             trigger(identificationNumberFieldName); 
         }
+        if (watchIdentificationType === IdentificationTypesCodes.PALESTINE_ID || watchIdentificationType === IdentificationTypesCodes.ID) {
```
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx:58]: same

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx [hunk 3]
```diff
@@ -38,6 +39,8 @@ const ContactForm: React.FC<Props> = ({ updatedContactIndex, contactStatus, pers
 
     const { shouldDisableContact } = useStatusUtils();
 
+    const [isId, setIsId] =  useState<boolean>(false);
```
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx:42]: remove one space after =

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningPersonal.tsx [hunk 4]
```diff
@@ -40,6 +41,7 @@ const ContactQuestioningPersonal: React.FC<Props> = (props: Props): JSX.Element
 
     const [shouldIdDisable, setShouldIdDisable] = useState<boolean>(false);
     const [age, setAge] = useState<string>(calcAge(interactedContact.birthDate));
+    const [isId, setIsId] =  useState<boolean>(false);
```
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningPersonal.tsx:44]: remove one space after =

**Review by chenFriedman — APPROVED** [review]

**Review by barnahum8 — APPROVED** [review]

**Review by chenFriedman — APPROVED** [review]

**Pull request merged**